src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,31 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
+    {
+        if (minValidations < 1)
+            minValidations = 1;
+
+        std::vector <std::uint32_t> times;
+
+        {
+            ScopedLockType sl (mLock);
+
+            auto j = findSet (hash);
+            for (auto& it : *j)
```

Comment:
I'm assuming that `findSet` returns a map with its contents already sorted in time sequence?  Otherwise your median calculation on line 412 doesn't make sense to me.  It seems counter-intuitive that the returned map is sorted in time order since you're calling `getSignTime()` on the value part of the map iterator, not the key.  But you know the code better than I do.